dist/webtorrent.chromeapp.js.LICENSE.txt

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Can we not generate these LICENSE files?

Reply:
is that.... a good idea?[ I don't really know much about copyright](https://github.com/ThaUnknown/jassub/issues/11) so I left it in

Reply:
We've never had it, never had any issues, we can include them once some copyright nerd requires them (?)

Reply:
maybe I could generate only one license file instead of both?

Reply:
@DiegoRBaquero done